package.json

```diff
@@ -5,7 +5,8 @@
   "dependencies": {
     "react": "^16.4.0",
     "react-dom": "^16.4.0",
-    "react-scripts": "1.1.4"
+    "react-scripts": "1.1.4",
+    "react-simple-storage": "^1.4.1"
```

Comment:
Wanted to mention I _did_ handle the `localStorage` manually before importing this dependency. It just makes the code a lot cleaner doing it this way. 😺